build.gradle

```diff
@@ -18,7 +18,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.2'
+        classpath 'com.android.tools.build:gradle:4.1.3'
```

Comment:
There was a bug in the current android gradle build tools which meant that the jacoco reports never waited for the android connected tests to actually run, making them always fail the build. Updating fixed the issue, but it also meant [this was needed to update too](https://github.com/bitrise-io/trace-android-sdk/pull/178/files#r608816417)